src/App.tsx

```diff
@@ -134,27 +191,54 @@ const App = () => {
           anchor="right"
           open={open}
           classes={{
-            paper: classes.drawerPaper,
+              paper: classes.drawerPaper,
```

Comment:
get rid of extra indent so it lines up like it was before. Also get rid of the comma at the end, it isn't necessary since there are no other parameters